artemis-core-client/src/main/java/org/apache/activemq/artemis/utils/XMLUtil.java

```diff
@@ -323,7 +323,7 @@ public static double parseDouble(final Node elem) {
    public static void validate(final Node node, final String schemaFile) throws Exception {
       SchemaFactory factory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
 
-      Schema schema = factory.newSchema(findResource(schemaFile));
+      Schema schema = factory.newSchema(new URL(findResource(schemaFile).toURI().toASCIIString()));
```

Comment:
Please.. check ./artemis-distribution/src/test/scripts/validate-spaces.sh


Please add the failing pattern that you found on that test.. and run it after building the release.


validate-spaces.sh has been added to avoid such issues. if you hit one i would like to have the test updated.

Reply:
I have updated this test. I hope it is ok to have all special characters on the sigle line.